extensions/wikia/Recirculation/js/experiments/placement.js

```diff
@@ -130,7 +148,27 @@ require([
 		helper.loadData()
 			.then(view.render)
 			.then(view.setupTracking(experimentName))
-			.fail(function() {});
+			.fail(handleError);
+	}
+
+	function handleError() {
+		// If there is an error somewhere we render the control group with no tracking
+		if (errorHandled) {
```

Comment:
I'm a little bit lost in the error handling especially in case with Lateral both modules. I assume it's possible one of the modules errs and then we still fall back to the control group, but the other module still shows.

Unless there's a good reason (plenty of fails) I would just track the errors here and leave the users with empty modules instead :-)